core-bundle/src/EventListener/CsrfTokenCookieSubscriber.php

```diff
@@ -49,7 +49,13 @@ public function onKernelRequest(RequestEvent $event): void
             return;
         }
 
-        $this->tokenStorage->initialize($this->getTokensFromCookies($event->getRequest()->cookies));
+        $request = $event->getRequest();
+
+        if (false === $request->attributes->get('_token_check')) {
+            return;
+        }
+
+        $this->tokenStorage->initialize($this->getTokensFromCookies($request->cookies));
```

Comment:
```suggestion
        $this->tokenStorage->initialize($this->getTokensFromCookies($event->getRequest()->cookies));
```

I think we should only skip the response event, not the token storage initialization.

Reply:
Changed in faaa63dfa1172b9e39ee7659df837cd5ea4d6de5.